buildSrc/src/main/kotlin/qai.java-conventions.gradle.kts

```diff
@@ -0,0 +1,55 @@
+plugins {
```

Comment:
This script is where all of your common logic that you want shared between projects should go. The projects then receive this logic by applying the plugin with id("qai.java-conventions")

You can make as many convention plugins as you want.